README.md

```diff
@@ -23,38 +24,35 @@ metadata:
   name: my-secret-with-ovirt-credentials
 type: Opaque
 stringData:
-  config.yaml: |-
+  ovirt: |-
     apiUrl: "https://ovirt-engine.example.com:8443/ovirt-engine/api/"
     username: admin@internal # provided in the format of username@domain
     password: 123456
     insecure: true
 EOF
 ```
 
-## Create a config map that defines the resource mappings from oVirt to KubeVirt:
+## Create oVirt resource mappings that defines how oVirt resources are mapped to kubevirt:
 ```bash
 cat <<EOF | kubectl create -f -
-apiVersion: v1
-kind: ConfigMap
+apiVersion: v2v.kubevirt.io/v1alpha1
+kind: ResourceMapping
 metadata:
-  name: ovirt-mapping-example
-  namespace: default
-data:
-  mappings: |-
-    networkMapping:
-      - source: red # maps of ovirt logic network to network attachment definition
-        target: xyz
-        type: bridge
-      - source: ovirtmgmt
-        target: pod
+  name: example-resourcemappings
+  namespace: example-ns
+spec:
+  ovirt:
+    networkMappings:
```

Comment:
I wonder whether it is enough to identify storagedomain or network by name.

Reply:
The definition of the source is based on https://github.com/kubevirt/vm-import-operator/pull/9/files#diff-7b282a764b7b115b17d31e0cdae066caR33 which accepts also `ID`.

If we have the origin VM, we can get its cluster and the data-center ID and using the resource name (network name or storage domain name) to uniquely identify it.